### PR TITLE
Fix #7622: Allow `yii\data\ArrayDataProvider` to control the sort flags

### LINKED ISSUE #7622: ArrayDataProvider is missing a property to control the sort flags for sortModels()
Could add a property

```
    /**
     * @var int sort flags to be used by [[sortModels()]]. This will be passed as the `` parameter
     * to [[ArrayHelper::multisort()]].
     */
    public $sortFlags = SORT_REGULAR;
```

or handle it via Sort class defined in `$attributes` property.

### PR DESCRIPTION
**Pull request by askobara**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  | #7622

### CODE REVIEW

**Comment by samdark** [comment]
OK. That one is looking clear and doesn't break anything.

File: framework/data/Sort.php [hunk 1]
```diff
@@ -186,6 +186,11 @@ class Sort extends BaseObject
      */
     public $urlManager;
 
+    /**
+     * @var int Allow to control a value of the fourth parameter which will be
+     * passed to [[ArrayHelper::multisort()]]
```
Comment (samdark) [inline framework/data/Sort.php]: Need `@since` tag.

**Review by samdark — APPROVED** [review]

**Review by schmunk42 — APPROVED** [review]

File: framework/data/Sort.php (hunk 1 above)
Comment (askobara) [reply framework/data/Sort.php]: done

**Comment by samdark** [comment]
Thanks!

**Pull request merged**